command/deployment_status.go

```diff
@@ -198,6 +201,9 @@ func formatDeploymentGroups(d *api.Deployment, uuidLength int) string {
 		}
 	}
 
+	// Sort the task group names to get a reliable ordering
+	sort.Strings(tgNames)
```

Comment:
It might be a good idea to add an assertion in an existing test to validate this logic.